### PR TITLE
DDF-3651 Updates map-info component to hide when cursor is off map

### PR DESCRIPTION
**Pull request by andrewkfiedler**
#### What does this PR do?
 - When the cursor goes off the map, the map-info component now hides.
 - Updates the map-info to not move around as the coordinates change.
 - Updates the map-info component to use a monospace font.
 - Pulls in Inconsolata as the base monospace font.
 - Adds fallbacks for our font families.

#### Who is reviewing it? 
@Schachte 
@djblue 
@adimka 
@mackncheesiest 
@jlcsmith 
@rzwiefel 

#### How should this be tested? (List steps with links to updated documentation)
Go off the map and verify that the coordinates disappear.  Also verify that the coordinates don't move around when they change.  

#### What are the relevant tickets?
[DDF-3651](https://codice.atlassian.net/browse/DDF-3651)

#### Screenshots (if appropriate)
https://codice.atlassian.net/secure/attachment/18231/map-info.mp4

### CODE REVIEW

**Comment by rymach** [comment]
Tested in:
- Firefox 
- Chrome
- Edge (Windows 10 VM)
- IE 11 (Windows 10 VM)
  - Baseline: coordinates show "- -" on the 3D map 
  - PR: the coordinate tooltip does not appear

**Comment by jlcsmith** [comment]
build now

**Comment by jlcsmith** [comment]
build now

File: ui/packages/catalog-ui-search/src/main/webapp/component/map-info/map-info.hbs [hunk 1]
```diff
@@ -12,7 +12,6 @@
  **/
  --}}
 <div class="info-feature">{{this.target}}</div>
-<div class="info-coordinates">
-    <span class="coordinate-lat">{{this.lat}}</span>
-    <span class="coordinate-lon">{{this.lon}}</span>
+<div class="info-coordinates"><!--
+    --><span class="coordinate-lat">{{this.lat}}</span> <span class="coordinate-lon">{{this.lon}}</span>
```
Comment (bdeining) [inline ui/packages/catalog-ui-search/src/main/webapp/component/map-info/map-info.hbs]: ✏️  Why is there an empty comment in here?

Comment (mackncheesiest) [reply ui/packages/catalog-ui-search/src/main/webapp/component/map-info/map-info.hbs]: I believe it's related to getting things to render in the browser in the way we want them. It's essentially commenting out the newline.

**Pull request merged**